core/src/test/java/io/grpc/internal/AbstractServerStreamTest.java

```diff
@@ -68,6 +74,22 @@ public WritableBuffer allocate(int capacityHint) {
   @Test
   public void frameShouldBeIgnoredAfterDeframerClosed() {
     ServerStreamListener streamListener = mock(ServerStreamListener.class);
```

Comment:
Done. This would be a nicer approach in the other tests as well, but I'd prefer to hold off on that as a follow-up PR, after everything else is solidified.